pages/windows/repair-bde.md

```diff
@@ -0,0 +1,35 @@
+# repair-bde
+
+> Attempt to repair or decrypt a damaged BitLocker-encrypted volume.
+
+- Attempt to repair a specified volume:
+
+`repair-bde {{input_volume}}`
```

Comment:
What does the input_volume and output_volume look like ? Can we give an example ?

Reply:
Input and output volumes can have 2 formats:
- the drive letter with a colon. (e.g. `C:`)
- the id of the volume (e.g. `\\?\Volume{id}\`)

Reply:
I think going with a drive letter will make things clearer.

Reply:
Sounds good. :+1: